README.md

```diff
@@ -44,6 +44,14 @@ Spyder may also be used as a PyQt4/PySide extension library (module
 `spyderlib`). For example, the Python interactive shell widget used in
 Spyder may be embedded in your own PyQt4/PySide application.
 
+## Running from source
```

Comment:
If there is user level docs, it is possible to add a redirect to them at the top of README.md The top also needs a link to wiki KB.